govcd/vm.go

```diff
@@ -1501,3 +1501,38 @@ func (client *Client) QueryVmList(filter types.VmQueryFilter) ([]*types.QueryRes
 	}
 	return vmList, nil
 }
+
+// UpdateVmCapabilities updates VM Capabilities and returns refreshed VM or error.
+func (vm *VM) UpdateVmCapabilities(cpuHot, memoryHot bool) (*VM, error) {
```

Comment:
IMO we are saving little characters but having such var names would be easier to comprehend:
```suggestion
func (vm *VM) UpdateVmCapabilities(cpuHotPlug, memoryHotPlug bool) (*VM, error) {
```

Reply:
I don't like plug. 
Agree to change if needed to:
func (vm *VM) UpdateVmCapabilities(cpuHotAdd, memoryHotAdd bool) (*VM, error) {

Reply:
Works for me as well. Let's see what others say. 
cc @dataclouder , @lvirbalas

Reply:
renamed